packages/evolution-common/package.json

```diff
@@ -24,6 +24,7 @@
     "chaire-lib-common": "^0.2.2",
     "geojson-validation": "^1.0.2",
     "i18next": "^22.4.15",
+    "libphonenumber-js": "^1.10.47",
```

Comment:
Do we really need this 8MB library or is the `evolution-common/src/utils/helper.ts` 's isPhoneNumber enough?

Reply:
isPhoneNUmber does not format phone numbers. I want to be able to format all of them using the international format. It should not be more than 145K minified